index.html

```diff
@@ -0,0 +1,32 @@
+<html>
+  <head>
+    <title>this is title</title>
+    <link rel="stylesheet" href="mystyle.css">
+  </head>
+  <body>
+    <div>
+    <p>
+      <span class="ex">
+        This is a <br>
+      </span>
+      <span class="warning">
+        awefwe
+      </span>
+      
+    <p>
+    </div>
+    <p class="nice" target="_hi">
```

Comment:
The `target` attribute is typically just used for `<a>` tags.  I don't usually see them in `<p>` tags.